packages/ui/src/message-room-view-header/MessageRoomViewHeader.jsx

```diff
@@ -22,10 +22,12 @@ export default function MessageRoomViewHeader(props) {
       <div className={styles.plugin__header}>
         <div
           onClick={() => {
-            if (props.roomInfo) {
```

Comment:
Why did you remove the if block? Is the room info not being received on the frontend?

Reply:
Yes, it's not...it's returning undefined

Reply:
Okay. I'll look into that